.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: /go/src/github.com/argoproj/argo-rollouts
+    docker:
+      - image: circleci/golang:1.12.6
```

Comment:
@jessesuen can you comment on upgrading to 1.12.6 in this PR?

Reply:
I run with 1.12.5 so I think this should be fine